dev-tools/ls-apis/api-manifest.toml

```diff
@@ -0,0 +1,420 @@
+# Describes all the Dropshot/OpenAPI/Progenitor APIs in the system
+#
+# See README.adoc in this package for details.
```

Comment:
Wow, this is incredibly detailed and clear documentation. Thanks for doing this! I think all the individual bits make sense.